frontend/src/app/application-forms/temporary-outfitters/temporary-outfitters.component.html

```diff
@@ -79,15 +79,18 @@ <h3 id="good-standing-evidence-label">{{ goodStandingEvidenceMessage }} <span cl
             [pointOfView]="pointOfView"
             (updateRootDateStatus)="updateDateStatus($event)"></app-activity-description>
 
-          <label id="location-map-label" class="usa-input">Map of proposed activity</label>
-          <p id="location-map-hint-text" class="help-text usa-form-hint">In addition to the description provided above, you may also include a map for further clarification of the specific areas you will be using within the forest.</p>
+          <legend id="location-map-label" class="usa-input location-map-url">Map of proposed activity</legend>
```

Comment:
What's the reason for switching from `<label>` to `<legend>` here?

Reply:
I'm not sure why I did that. I went ahead and reverted to <label>.